vec/vec-v113/src/main/resources/generate-xjb-from-vec-model.xsl

```diff
@@ -17,7 +17,7 @@
     <xsl:key name="parents" match="ownedAttribute[@xmi:type='uml:Property' and exists(@association) and @aggregation='composite']" use="@type"/>
     <xsl:key name="incoming-refs" match="ownedAttribute[@xmi:type='uml:Property' and exists(@association) and not(@aggregation='composite')]" use="@type"/>
 
-    <xsl:variable name="VEC_VERSION">1.1.3</xsl:variable>
+    <xsl:variable name="VEC_VERSION">2.0.2</xsl:variable>
```

Comment:
You did change the wrong xsl file. This is the one under vec-v113. Please copy the original one to the vec-v2x package and make the required changes there.

Reply:
Because the 2X project hadn't its own before I started the PR. It is the same file for all versions (except the "version" variable). The main point why I changed it, is that somebody xml-formatted it wrongly, resulting in generated attribute values having line brakes and whitespace, making it malformed XML...

Anyway 2x requires a completely new one, when we start extracting meta information  from the xsd instead of the XMI model.

Reply:
Ok I see, but than still the VEC_VERSION variable in that xsl should remain 1.1.3, right?

Reply:
Or we extract it outside the specific vec packages. It does not make sence to have it in the resource-folder anyway since it should not be packages with the arty.

Reply:
I would prefer that solution. It is a "one time" job for each version. It is a pure utility xsl, not used in the build process, and you are right, it should not be packaged for runtime.

Reply:
Then please move it out. Maybe a script folder? And the same should be done for the KBL side of things.

Reply:
Done